editor/src/consts.rs

```diff
@@ -27,8 +27,9 @@ pub const SCALE_SNAP_INTERVAL: f64 = 0.1;
 pub const SLOWING_DIVISOR: f64 = 10.;
 
 // Select tool
-pub const SELECTION_TOLERANCE: f64 = 1.;
+pub const SELECTION_TOLERANCE: f64 = 5.;
 pub const SELECTION_DRAG_ANGLE: f64 = 90.;
+pub const SELECTION_HOVER_WEIGHT: f64 = 3.;
```

Comment:
I think `2` would be good, `3` is a bit chonky.

Reply:
`2` is a good weight now 👍. But it's a different weight from the Path Tool's outline, which seems to be a little thinner. Can you please rename this to `PATH_OUTLINE_WEIGHT` and make the Path Tool's outline also use it?